tests/pmv_test.py

```diff
@@ -15,171 +21,61 @@ def tearDown(self):
         """Nothing to tear down as nothing gets written to file."""
         pass
 
-    def test_default_values(self):
-        """Test the default values being correctly imported and calculated"""
-        pmv_comf = PMV.from_individual_values()
-
-        assert pmv_comf.air_temperature == [20]
-        assert pmv_comf.rad_temperature == [20]
-        assert pmv_comf.wind_speed == [0]
-        assert pmv_comf.rel_humidity == [50]
-        assert pmv_comf.met_rate == [1.1]
-        assert pmv_comf.clo_value == [0.85]
-        assert pmv_comf.external_work == [0]
-        assert pmv_comf.ppd_comfort_thresh == 10
-        assert pmv_comf.humid_ratio_up == 0.03
-        assert pmv_comf.humid_ratio_low == 0
-        assert pmv_comf.still_air_threshold == 0.1
-        assert pmv_comf.single_values is True
-
-        pmv = pmv_comf.pmv
-        # Assert similarity to 0.01 decimal based of http://comfort.cbe.berkeley.edu/
-        assert round(pmv - (-0.85), 2) == 0
-
-        ppd = pmv_comf.ppd
-        # Assert similarity to 0.01 decimal based of http://comfort.cbe.berkeley.edu/
-        # Remember that the values are in %
-        assert round(ppd - 20) == 0
-
-        # Check pmv, set and cooling effect are calculated properly
-        # based of http://comfort.cbe.berkeley.edu/
-        pmv = pmv_comf.pmv
-        ppd = pmv_comf.ppd
-        set = pmv_comf.set
-        cooling_effect = pmv_comf.cooling_effect
-
-        assert round(pmv - (-0.85), 2) == 0
-        assert round(ppd - 20) == 0
-        assert round(set - 22.2, 1) == 0
-        assert round(cooling_effect - 0, 1) == 0
-
-    def test_setters_single_values(self):
-        """Test low level single value setting functions"""
-        pmv_comf = PMV.from_individual_values()
-        air_temp = 19
-        rad_temp = 23
-        wind_speed = 0.5
-        rel_humid = 60
-        met_rate = 1.5
-        clo = 0.4
-        external_work = 0
+    def test_pmv_fanger_utility(self):
+        """Test the pmv_fanger utility function"""
+        pmv_comf, ppd, hl = pmv_fanger(19, 23, 0.1, 60, 1.5, 0.4)
+        assert pmv_comf == pytest.approx(-0.680633, rel=1e-2)
+        assert ppd == pytest.approx(14.7373, rel=1e-2)
```

Comment:
Why not checking the value for `hl`?

Reply:
Well, let's first decide if I have to take it out based on comments above.